plutus-ledger/src/Ledger/Orphans.hs

```diff
@@ -109,15 +108,11 @@ instance OpenApi.ToSchema JSON.Value where
     declareNamedSchema _ = pure $ OpenApi.NamedSchema (Just "JSON") mempty
 deriving instance OpenApi.ToSchema ann => OpenApi.ToSchema (Kind ann)
 deriving newtype instance OpenApi.ToSchema Ada
-deriving instance OpenApi.ToSchema Tx
 deriving instance OpenApi.ToSchema ScriptTag
 deriving instance OpenApi.ToSchema RedeemerPtr
 deriving instance OpenApi.ToSchema TxOutRef
-deriving instance OpenApi.ToSchema TxInType
-deriving instance OpenApi.ToSchema TxIn
```

Comment:
These shouldn't be removed

Reply:
We now own the definition of TxInType and TxIn so they are not orphan instances anymore.